### ISSUE TITLE
include.js, embedded winchan needs updating, breaks IE9 testing on localhost

### ISSUE THREAD

**Issue by mmullens** [mozilla/persona#1199]
resources/static/include_js/include.js:

does not embed the latest version of winchan which causes inability to find relay frame while developing browserid enabled site on localhost and testing on localhost with IE9 (and probably others).  Issue is that the relay frame is never found.  

The fix is to embed the latest version of winchan in include.js.  There are some significant fixes in latest winchan that should be kept in sync with include.js since it is being embedded.

Thanks,
Matt

**Comment by shane-tomlinson** [comment]
@mmullens - is this still an issue?

**Comment by mmullens** [comment]
It is difficult for me to check at the moment, but I think that it is.  I checked about a month or so ago and there were differences in findRelay and somewhere in the onOpen method if memory serves.  Basically I took winchan out of include.js and compared it to winchan on github, found the differences, patched it, and then things worked.  I apologize that I can't be of more help right now, I should have uploaded the diff while I had it for ease of investigation.

But yes, if it is generated, then that is much easier if you expect many changes to winchan in the future.  Thank you for all of your work on this system, beautiful piece of work!
